src/components/Tabs/Tabs.module.css

```diff
@@ -5,12 +5,57 @@
 \*------------------------------------*/
 
 /**
- * 1) List of of links where each link toggles open associated information
+ * List of of links where each link toggles open associated information
  */
-/* stylelint-disable-next-line block-no-empty */
 .tabs {
+  position: relative;
 }
 
+/**
+ * Fade scrollable indicators to display if there is scrollable area on either side.
+ * 1) Indicators are non interactable and should be able to click tab under them.
+ */
+.tabs--scrollable-left::before,
+.tabs--scrollable-right::after {
+  pointer-events: none; /* 1 */
+  content: '';
+  position: absolute;
+  top: 0;
+  width: var(--eds-size-8);
+
+  height: 2.75rem;
```

Comment:
Height needs to be fixed since `content: ''`. Alternative is to have a `calc(padding + text-size * line-height multiplier)` but 1) text sizes and line-height multipliers are not css variables/tokens anyways and 2) figured that would be more confusing than having fixed height.

Reply:
Is it possible to have `--current-font-size` and `--current-line-height` declared within typography tokens for use in cases like this, or would that be too niche?